cluster_management/src/main/java/com/pinterest/rocksplicator/shardmap/ResourceMap.java

```diff
@@ -0,0 +1,24 @@
+package com.pinterest.rocksplicator.shardmap;
+
+import java.util.List;
+import java.util.Set;
+import javax.swing.Painter;
```

Comment:
why do we nee Painter here?

Reply:
My bad. IntelliJ auto import mistake. Will fix this.